src/main/webapp/3Dscript.js

```diff
@@ -194,8 +199,39 @@ function gui() {
   gui.add(poseTransform, 'scale', .5, 2, .25)
       .onChange(plotOrientation)
       .onFinishChange(plotTrajectory).name('Pose Scale Multiplier');
+  /* Time value. */
+  const timeStart = {time: 'no point yet'};
```

Comment:
Can you just leave the uninitialized fields unset? It's surprising to have fields switch between holding a string and a number.